client/i18n/locales/english/intro.json

```
@@ -2900,6 +2900,18 @@
           "You'll practice using loops and conditionals to calculate the factorial of a number."
         ]
       },
+      "lab-mutations": {
+        "title": "Mutations Algorithm",
```

```suggestion
        "title": "Implement the Mutations Algorithm",
```
I guess "Implement" could work for both labs